pkg/controllers/cassandra/cluster_control.go

```diff
@@ -3,19 +3,20 @@ package cassandra
 import (
 	"github.com/golang/glog"
 	v1alpha1 "github.com/jetstack-experimental/navigator/pkg/apis/navigator/v1alpha1"
+	"github.com/jetstack-experimental/navigator/pkg/controllers/cassandra/service"
 	"k8s.io/client-go/tools/record"
 )
 
 const (
-	errorSync = "ErrSync"
+	ErrorSync = "ErrSync"
 
-	successSync = "SuccessSync"
+	SuccessSync = "SuccessSync"
 
-	messageErrorSyncServiceAccount = "Error syncing service account: %s"
-	messageErrorSyncConfigMap      = "Error syncing config map: %s"
-	messageErrorSyncService        = "Error syncing service: %s"
-	messageErrorSyncNodePools      = "Error syncing node pools: %s"
-	messageSuccessSync             = "Successfully synced CassandraCluster"
+	MessageErrorSyncServiceAccount = "Error syncing service account: %s"
+	MessageErrorSyncConfigMap      = "Error syncing config map: %s"
+	MessageErrorSyncService        = "Error syncing service: %s"
+	MessageErrorSyncNodePools      = "Error syncing node pools: %s"
+	MessageSuccessSync             = "Successfully synced CassandraCluster"
```

Comment:
Is there a reason for exporting these? I'm not against it, just wondering.

Reply:
I made them public so that I can look for the sync success or failure messages in the fake event recorded....to know when the Sync operation has completed in the  integration test in cassandra_test.go.